src/main/msc/emfat.c

```diff
@@ -390,7 +390,7 @@ void read_boot_sector(const emfat_t *emfat, uint8_t *sect)
     bs->volume_id[1] = 14;
     bs->volume_id[2] = 13;
     bs->volume_id[3] = 8;
-    memcpy(bs->volume_label, "NO NAME     ", 12);
+    memcpy(bs->volume_label, "NO NAME    ", 11);
```

Comment:
Should really be using `VOL_LABEL_LEN` here.

Reply:
I also think it is better to use here VOL_LABEL_LEN and also in the next line
memcpy(bs->file_system_type, "FAT32   ", FILE_SYS_TYPE_LENGTH);
I will fix it.